devices/develco/aqszb-110_voc_sensor.json

```diff
@@ -3,8 +3,8 @@
   "uuid": "bf113ff9-a8c5-4e1e-9088-33cc0d5f4daf",
   "manufacturername": "Develco Products A/S",
   "modelid": "AQSZB-110",
-  "vendor": "Develco Products",
-  "product": "AQSZB-110 TVOC sensor",
```

Comment:
Is there any particular reason why you chop the vendor name here and add another vendor in the product name? That goes for all Develco devices.

Reply:
I did a web search and for me Frient it more a series name than a vendor. Maybe I am wrong ?

Reply:
@SwoopX can you please suggest an example which can be used to make these consistent for Develco devices?